assets/js/plugin-check-admin.js

```diff
@@ -12,7 +12,9 @@
 		e.preventDefault();
 
 		getChecksToRun()
+		.then( setupEnvironment )
 		.then( runChecks )
+		.then( cleanupEnvironment )
```

Comment:
See my comment below, rather than chaining these and thereby forcing every AJAX response to include its request parameters to pass it on to the next AJAX request, I think we should decouple them.

We can then change the JS functions to accept dedicated parameters for the things that each function needs rather than the generic `data` object from the previous AJAX response.

But like my comment above, let's handle this refactoring in a separate PR.